http/src/test/resources/reference.conf

```diff
@@ -92,6 +92,16 @@ pubsub {
       max-attempts = 10 # 10 seconds * 90 is 15 min
       interval = 2 seconds
     }
+    createApp {
+      interval = 0 seconds
+      max-attempts = 120
+      interruptAfter = 30 seconds
+    }
+    deleteApp {
+      initial-delay = 0 seconds
+      interval = 0 seconds
+      max-attempts = 120
+    }
```

Comment:
reducing the delays and intervals to 0 in tests reduces test time by 5 minutes

Reply:
thank you! I noticed the AKS/GKE tests take a while